src/app/pages/Designathons/Designathon24/components/WinnerShowcase/WinnerShowcase.module.scss

```diff
@@ -0,0 +1,95 @@
+.container {
+	background-color: var(--black);
+	color: white;
+
+	& .heading {
+		font-weight: 700;
+		color: var(--peach);
+	}
+
+	& .winnerContainer {
```

Comment:
Thought [repeating]: since we are already nesting with SASS, I don't think the parent selectors and child combinators are necessary?

Reply:
In [db078d8](https://github.com/designatuci/DUCI-website/pull/52/commits/db078d8a28bfcb6debe374cf973186a1bf57d655), I remove the `& .heading` selector.

Let me know if I misinterpreted what you meant -- (am new to SASS conventions)

Reply:
No, I was saying that every nested selector in this entire file shouldn't need the `&` since SASS will combine the parents accordingly.